bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2OiwfsGuideProbe.java

```diff
@@ -178,7 +175,4 @@ public VignettingCalculator calculator(ObsContext ctx) {
                 F2OiwfsProbeArm$.MODULE$,
                 F2ScienceAreaGeometry$.MODULE$);
     }
-
-    @Override
-    public BandsList getBands() { return RBandsList.instance(); }
```

Comment:
Since most of the probes have this exact method definition for `getBands`, I moved this as the `default` implementation in the `GuideProbe` class to clean up repeated code.